### ISSUE TITLE
Name ideas

### ISSUE THREAD

**Issue by davidanthoff** [julia-vscode/julia-vscode#1528]
Post your ideas for a name here!

**Comment by goretkin** [comment]
Juno!

**Comment by oysteinsolheim** [comment]
JuCo!

**Comment by davpoolechem** [comment]
VSJuno

**Comment by abelsiqueira** [comment]
JanVS (Janus) - "Braden Eliason: JanVS - U's were often written as V in Latin"

**Comment by versipellis** [comment]
Abel Siqueira suggested Janus in the Discord. A variation of it by Braden Eliason that got a lot of upvotes is "JanVS", with the U as V in Latin.

**Comment by BruceAyati** [comment]
The fruit Jujube

**Comment by mbauman** [comment]
Janus is clever and could be styled as Jᴀɴᴠs.

**Comment by lewisl** [comment]
Hera

**Comment by kdheepak** [comment]
I'd like to throw [Vesta](https://en.wikipedia.org/wiki/Vesta_(mythology)) into consideration, being the sibling of [Juno](https://en.wikipedia.org/wiki/Juno_(mythology)) :)

**Comment by EricForgy** [comment]
> Juno!

I kind of like calling it "Juno" too, but as described by David on Slack:

> We seriously thought about keeping the Juno name, but in the end it felt very much like Microsoft's naming strategy from the early 2000s: use one name (".Net") for everything you do, and in the end no one knows anymore what it actually stands for. "Juno" right now has a very well defined meaning in the Julia ecosystem, and it seems it would just cause a lot of confusion for folks if it now all of a sudden referred to something else.
>
> Just think of the hundreds if not thousands of existing posts on forums/github etc that all use "Juno" to refer to the Atom integration.

> VSJuno

I like that, but maybe "VS Juno"? I think this is a nice homage to "Juno" while avoiding the potential confusion that David is (rightly) concerned about.

**Comment by el-oso** [comment]
How about [Vulcan](https://en.wikipedia.org/wiki/Vulcan_(mythology))?

**Comment by tk3369** [comment]
Juicy!

**Comment by naivej** [comment]
Judio (Julia studio)?

**Comment by mortenpi** [comment]
`junocode` (Junocode? JunoCode?)

**Comment by mileslucas** [comment]
> How about Vulcan?

Might get confusing with the [Vulkan graphics library](https://en.wikipedia.org/wiki/Vulkan_(API))

**Comment by tsela** [comment]
Personally, I think we should stick to "Julia" or "Julia extension". All the main ("official") extensions for languages in vscode are simply named after the language itself (even the Go extension is named "Go" rather than "Golang"). I see no point in differentiating yourself here. It would only make the Julia extension look less "professional" than other language extensions.
If you really want to honour the Juno team, why not change the name of the organisation in vscode? Right now it's simply "julialang". You could make it whatever you want to, including one of the options here.

**Comment by mileslucas** [comment]
@tsela 
https://julialang.slack.com/archives/C6LV2R5JB/p1596081578327500
>David Anthoff  10:59 PM
We will not change the name of the extension itself in the marketplace, we would probably change the name of the publisher and call the overall effort this new name. We would also use that name in things like discourse forum etc. But the extension in the marketplace itself would still just be called Julia.

IIUC There aren't plans to change the marketplace name, this naming would be describing the whole endeavor, which is much more than a vscode extension (LanguageServer.jl, etc.).

**Comment by tsela** [comment]
> @tsela 
> https://julialang.slack.com/archives/C6LV2R5JB/p1596081578327500
> >David Anthoff  10:59 PM
> We will not change the name of the extension itself in the marketplace, we would probably change the name of the publisher and call the overall effort this new name. We would also use that name in things like discourse forum etc. But the extension in the marketplace itself would still just be called Julia.
> 
> IIUC There aren't plans to change the marketplace name, this naming would be describing the whole endeavor, which is much more than a vscode extension (LanguageServer.jl, etc.).

@mileslucas yeah I know, I just read that slack message about 2 minutes after I wrote that comment. Feeling a bit stupid now, as I wrote on slack 😅.

**Comment by cdsousa** [comment]
Julide

**Comment by Bondan000** [comment]
how about 'Jules'?

**Comment by el-oso** [comment]
> Personally, I think we should stick to "Julia" or "Julia extension". All the main ("official") extensions for languages in vscode are simply named after the language itself (even the Go extension is named "Go" rather than "Golang"). I see no point in differentiating yourself here. It would only make the Julia extension look less "professional" than other language extensions.
> If you really want to honour the Juno team, why not change the name of the organisation in vscode? Right now it's simply "julialang". You could make it whatever you want to, including one of the options here.

Actually you are right ... for Atom it was a bit confusing for new people to distinguish what was Juno given that they were using Julia and Atom. Perhaps it can just simply be called "The Juno Team" which produces the "Julia plugin for VSCode"

**Comment by pb866** [comment]
Similar to some previous, just some slight variations:
- `JUDE`: JUlia Development Environment
- `JUICE`: Julia Integrated Coding Enviroment

**Comment by sosiristseng** [comment]
> How about [Vulcan](https://en.wikipedia.org/wiki/Vulcan_(mythology))?

It's a name for a [Graphical API](https://www.khronos.org/vulkan/).

**Comment by el-oso** [comment]
> > How about [Vulcan](https://en.wikipedia.org/wiki/Vulcan_(mythology))?
> 
> It's a name for a [Graphical API](https://www.khronos.org/vulkan/).

_Incorrect_ the name of the API is Vul**k**an with a K. In my opinion no confusion because they are completely different things.

In Greek mythonlogy Vulcan is the son of Juno, the god of fire and metalworking.

**Comment by rockper** [comment]
Junebug. Simple, unique, easy on the ear, fully pronounced word (good for presentations) - with a minor hat tip to Juno.

**Comment by StefanKarpinski** [comment]
If the VS Code plugin is going be called "Julia" and the repo for the plugin is https://github.com/julia-vscode/julia-vscode then what is the additional name for? Where would such a name appear?
